### ISSUE TITLE
[Feature Request] Display title (additional templated title field)

### ISSUE THREAD

**Issue by ChillingPepper** [yt-dlp/yt-dlp#1052]
## Checklist

- [x] I'm reporting a feature request
- [x] I've verified that I'm running yt-dlp version **2021.09.02**
- [x] I've searched the bugtracker for similar feature requests including closed ones

## Description

Many media players can use yt-dlp to play media from yt-dlp supported sites. For example mpv, where it is as simple as `mpv https://youtu.be/whatever`. These integrations use the JSON interface to fetch the formats for playback and use the provided title in their GUIs. Some sites would benefit a lot from a customizable title.

For example streamlink, a "yt-dlp for live Twitch streams" sets the media player title to `{channel} - {game} - {stream_title}`.
For sites that offer TV shows or similar episodic content the media player title could be `{show} {season_number}x{episode_number} - {episode_title}`.

So I'm proposing a new field: `display_title`. This field can use the same template as the filename templates.
The global default is the same as `title`: `%(title)s`, but this can be customized in each extractor and by the user through an option.

To summarize: the existing `title` field stays as it is but a new `display_title` filed is added that can be templated like the filename output templates.

**Comment by pukkandan** [comment]
You can already modify any metadata field. Look at `--parse-metadata` and the parsing metadata section of readme

**Comment by ChillingPepper** [comment]
That's perfect, thank you!
